runtime/oti/j9.h

```diff
@@ -367,6 +367,8 @@ static const struct { \
 #define IS_JAVA_LANG_THREAD(vmThread, object) \
 	isSameOrSuperClassOf(J9VMJAVALANGTHREAD_OR_NULL((vmThread)->javaVM), J9OBJECT_CLAZZ(vmThread, object))
 
+#define J9VM_VIRTUALTHREAD_ROOT_NODE_STATE ((I_32) 0xBAADF00D)
```

Comment:
space is not needed.
```suggestion
#define J9VM_VIRTUALTHREAD_ROOT_NODE_STATE ((I_32)0xBAADF00D)
```